gap/matrix/slconstr.gi

```diff
@@ -2466,56 +2466,52 @@ end;
 #! @BeginChunk NaturalSL
 #! TODO
 #! @EndChunk
-FindHomMethodsMatrix.NaturalSL := function(ri,G)
-  local data,e,genlist,grpmem,i,j,n,p,q,r,il;
-
+FindHomMethodsMatrix.NaturalSL := function(ri, G)
+    local data, e, genlist, grpmem, i, j, n, p, q, r, il;
     if not IsMatrixGroup(G) then
         return NeverApplicable;
     fi;
-
-    il := InfoLevel(InfoMethSel);
-    SetInfoLevel(InfoMethSel,0);
-    r:=RecogniseClassical(G,"linear");
-    SetInfoLevel(InfoMethSel,il);
+    r := RecogniseClassical(G, "linear");
```

Comment:
Again a change that potential is not a "clean up". Well, it would be if you didn't touch `RecogniseClassical` as that already messes with `InfoMethSel` in the same way...